AirCasting/SessionViews/MultiSliderView.swift

```diff
@@ -26,26 +35,24 @@ struct MultiSliderView: View {
     var thresholdVeryLow: Float {
         thresholds.first ?? 0
     }
+        
     var colors: [Color] = [Color.aircastingGreen, Color.aircastingYellow, Color.aircastingOrange, Color.aircastingRed]
         
     var body: some View {
         GeometryReader { geometry in
-            let frameWidth = geometry.frame(in: .local).size.width
-            
             ZStack {
                 colors.last
                 ForEach(thresholdButtonValues.indices.reversed(), id: \.self) { index in
                     colors[index]
-                        .frame(width: CGFloat(thresholdButtonValues[index]) * frameWidth / CGFloat(thresholdVeryHigh))
-                        .position(x: CGFloat(thresholdButtonValues[index]) * frameWidth / CGFloat(thresholdVeryHigh) / 2,
+                    // TODO: handle situation when thresholdVeryHigh = 0
```

Comment:
Please use `#warning`

Reply:
It's super weird.
When I was adding a `#warning` there, I was getting an error that there's too much operation and the compiler cannot compile in a reasonable time. I wanted to paste a screenshot to prove it but it doesn't appear anymore. 
So I changed comment to warning.